### PR TITLE
Atoms/Icons -> typescript

### PR DESCRIPTION
**Pull request by ortizlu**
Converted all of the icons to typescript, updated all tests to use testing-library-react, and updated all of the pathnames. 

make sure conversion is 1 to 1 for all components.

N/A

### CODE REVIEW

File: components/atoms/icons/ArrowEllipsisIcon/ArrowEllipsisIcon.tsx [hunk 1]
```diff
@@ -14,7 +14,10 @@ type ArrowEllipsisIconProps = {
   style?: React.CSSProperties;
 };
 
-/** Renders an arrow ellipsis icon. */
+/** Displays the ArrowEllipsisIcon component.
+ * @param props - Component props.
```
Comment (JamesIves) [inline components/atoms/icons/ArrowEllipsisIcon/ArrowEllipsisIcon.tsx]: I'm not sure these comments are really necessary here as we annotate the prop types.

Comment (RohitRajendran) [reply components/atoms/icons/ArrowEllipsisIcon/ArrowEllipsisIcon.tsx]: Yeah agreed!

Comment (CalMlynarczyk) [reply components/atoms/icons/ArrowEllipsisIcon/ArrowEllipsisIcon.tsx]: Same

Comment (ortizlu) [reply components/atoms/icons/ArrowEllipsisIcon/ArrowEllipsisIcon.tsx]: Do we still want the `@returns` comments or should we just keep the description of the function only

Comment (JamesIves) [reply components/atoms/icons/ArrowEllipsisIcon/ArrowEllipsisIcon.tsx]: I think just the description

**Pull request merged**